netlify/functions/messages.js

```
@@ -66,7 +66,8 @@ export const handler = async (event) => {
 
       case "POST":
         // Send a new message
-        const { username, message } = JSON.parse(body);
+        const { username, message, replyToId, replyToUsername, replyPreview } =
+          JSON.parse(body);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)